howdoi/howdoi.py

```diff
@@ -372,6 +376,25 @@ def _clear_cache():
     return cache.clear()
 
 
+def _is_help_query(query: str):
+    return any(
+            [   query.lower() == help_query for help_query in SUPPORTED_HELP_QUERIES]
+            )
+
+
+def _get_help_instructions():
+    instruction_splitter = build_splitter(' ',60)
+    answer = [
+        'Here are a few popular howdoi commands ',
+        '>>> howdoi foo (default query)',
```

Comment:
rather than `foo`, let's use a real query so folks can follow along at home :) Ideally something that has multiple good answers so -n3 will work well.